prune-releases.sh

```diff
@@ -17,11 +19,24 @@ function usage()
     echo "Example: $0 --older-than=\"2 weeks ago\" --helm-release-filter=\"^feature-.+-web$\" --namespace-filter=\"^feature-.+\""
 }
 
+function setup_helm() {
+    if [ ${HELM_VERSION} == "" ]; then
+        HELM_VERSION="v2.13.1"
+    fi
+
+    _get_helm "${HELM_VERSION}"
+    _set_helm_version "${HELM_VERSION}"
```

Comment:
For security, we don't want to be downloading helm releases at runtime. The helm chart defines a read-only filesystem so to support multiple helm releases they'll all have gathered during the image build process and the switching logic will have to be based on filename or something.